cmd/ctr/fetch.go

```diff
@@ -44,21 +44,19 @@ Most of this is experimental and there are few leaps to make this work.`,
 		var (
 			ref = clicontext.Args().First()
 		)
-		ctx, cancel := appContext(clicontext)
-		defer cancel()
-
-		_, err := fetch(ctx, ref, clicontext)
+		_, err := fetch(ref, clicontext)
```

Comment:
do we want to change the naming here from `clicontext` --> `context`?

Reply:
this file had context imported and used already so i kept with the same naming as most of the other files when this happens